docs/kf.dev/content/en/docs/general-info/kf-cli/commands/kf_push.md

```diff
@@ -27,9 +27,12 @@ kf push APP_NAME [flags]
 ### Options
 
 ```
+      --args stringArray            Overwrite the args for the image.
   -b, --buildpack string            Skip the 'detect' buildpack step and use the given name.
+  -c, --command string              Startup command for the app, this overrides the default command specified by the web process.
       --container-registry string   Container registry to push sources to. Required for buildpack builds not targeting a Kf space.
       --docker-image string         Docker image to deploy.
+      --entrypoint string           Overwrite the default entrypoint of the image.
```

Comment:
and `entrypoint` flag will override `command` flag, right?

Reply:
I added some additional text like you recommended. The command will also fail if you try to use the two together.